vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -67,6 +68,8 @@ def dash_ag_grid(data_frame, **kwargs):
                 "buttons": ["apply", "reset"],
                 "closeOnApply": True,
             },
+            "flex": 1,
```

Comment:
Regarding your comment on column size:
> I think the current behaviour works nicely! Just one question @maxschulz-COL - in the example above you added as a screenshot, I would have expected a horizontal scrollbar to appear automatically. Is this happening when there are too many columns? In general, I think the expected behaviour should be:

> - Columns should optimally always fit their content

This is as far as I understand linked to the property [`columnSize`](https://dash.plotly.com/dash-ag-grid/column-sizing#columnsize-property). The default is that the columns fit their content

> - If there are too many columns and their contents gets cut off -> horizontal scrollbar should appear

This is a consequence of the above default

> - When a user re-sizes the window, the Table should resize as well

We chose the option [`flex`](https://dash.plotly.com/dash-ag-grid/column-sizing#column-flex) for this - as it preserves column widths on resizing

As I understand, but please correct me if I am wrong @AnnMarieW, we can either:
- have columns size to their conent and a horizontal scroll bar appearing if there are too many/too long columns
OR
- have columns flexibly adjust to the table size, and preserve their size on table resizing

What I understand is not possible is that:
- columns fit to their content, but if its not enough to fill the entire grid, the columns get enlarged